onnxruntime/contrib_ops/cuda/bert/skip_layer_norm_impl.cu

```diff
@@ -69,44 +79,72 @@ __global__ void SkipLayerNormKernel(
   LayerNorm<T, TPB>(thread_data, ld, offset, beta, gamma, output);
 }
 
+#define InvokeSkipLayerNormKernelSmall(block_size, bias) \
+  if (bias != nullptr) {                                 \
+    SkipLayerNormKernelSmall<T, block_size, true>        \
+        <<<grid_size, block_size, 0, stream>>>(ld,       \
+                                               input,    \
+                                               skip,     \
+                                               beta,     \
+                                               gamma,    \
+                                               bias,     \
+                                               output);  \
+  } else {                                               \
+    SkipLayerNormKernelSmall<T, block_size, false>       \
+        <<<grid_size, block_size, 0, stream>>>(ld,       \
+                                               input,    \
+                                               skip,     \
+                                               beta,     \
+                                               gamma,    \
+                                               bias,     \
+                                               output);  \
+  }
+
 template <typename T>
-bool ComputeSkipLayerNorm(
-    cudaStream_t stream, const int ld, const int n, const T* input, const T* skip,
-    const T* beta, const T* gamma, const T* bias, T* output) {
+bool ComputeSkipLayerNorm(cudaStream_t stream,
+                          const int ld,
+                          const int n,
+                          const T* input,
+                          const T* skip,
+                          const T* beta,
+                          const T* gamma,
+                          const T* bias,
+                          T* output) {
   // this must be true because n is the total size of the tensor
   assert(n % ld == 0);
   const int grid_size = n / ld;
 
   if (ld <= 32) {
-    constexpr int block_size = 32;
-    SkipLayerNormKernelSmall<T, block_size>
-        <<<grid_size, block_size, 0, stream>>>(ld, input, skip, beta, gamma, bias, output);
+    InvokeSkipLayerNormKernelSmall(32, bias)
   } else if (ld <= 128) {
     constexpr int block_size = 128;
```

Comment:
The following lines can be removed:
   constexpr int block_size = 128; 
   constexpr int block_size = 384; 
and apply similar change for block_size=256